manticore/manticore.py

```diff
@@ -271,19 +296,12 @@ def _real_context():
                 yield ctx
                 context[key] = ctx
 
-    @property
-    def verbosity(self):
+    @staticmethod
+    def verbosity(level):
```

Comment:
Should this be `set_verbosity`?

Reply:
Perhaps, I personally find `Manticore.verbosity(2)` more aesthetically pleasing than `Manticore.set_verbosity(2)`, but it's a mild preference